### PR TITLE
Added needed functions

### PR DESCRIPTION
**Pull request by SebastianNeiswanger**
I'll be honest in that I don't know for sure if this will work. They are small functions that do as requested in the issue.

### CODE REVIEW

**Comment by SpBills** [comment]
Do you not have the ability to run the test environment?

**Review by SpBills — CHANGES_REQUESTED** [review]
Variable consistency. But good job otherwise.

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -20,7 +22,12 @@ export const Layout = ({ children }: PropsWithChildren) => {
     } 
   };
 
+  const get_reward_tiers = async () => {
+    set_tiers(await get_all_reward_tiers());
+  }
+
   useEffect(() => { do_login_check() }, []);
+  useEffect(() => { get_reward_tiers() }, []);
```
Comment (SpBills) [inline src/components/Layout.tsx:30]: we shouldn't have 2 use effects. just put it after the `do_login_check()`

**Review by SpBills — CHANGES_REQUESTED** [review]

File: src/components/Layout.tsx (hunk 1 above)
Comment (SebastianNeiswanger) [reply src/components/Layout.tsx:30]: ah I was following what we did in another file and didn't know for sure if we were able to double up like that

**Review by SpBills — APPROVED** [review]
Thanks!

**Pull request merged**